### ISSUE TITLE
Error: Dojo loader version does not match the version of Dojo.

### ISSUE THREAD

**Issue by soeik** [OpenNTF/dojo-webpack-plugin#213]
Could someone please help me to figure out why I'm getting this error:
`Error: Dojo loader version does not match the version of Dojo.
Loader version = 1.15.0.
Dojo version = 0.0.1.
`
Problem is that it takes Dojo version from my package.json, which doesn't seem to be a right place. 

If I change version in package.json - everything works fine.

Thanks in advance!

**Comment by chuckdumont** [comment]
The check is for preventing difficult to identify problems resulting from using mismatched versions of Dojo and the Dojo loader.  If you are specifying a pre-built loader using the [loader](https://github.com/OpenNTF/dojo-webpack-plugin#loader) option, then the version of the loader must match the version of Dojo that is used in the webpack build.  The loader version, as determined from the package.json at the time the loader was built, is embedded in the built loader when you run `dojo-webpack-plugin/buildDojo/build.js`.  When you build your application using webpack, and you specify the built loader using the `loader` option, the Dojo version embedded in the built loader is compared with the version of Dojo used by webpack (again, from the package.json) and if they don't match then that error is thrown.

**Comment by soeik** [comment]
Thanks for a fast response.

I understand purspose of this check, however I was curious why it takes dojo version from package.json of my application and not from `node_modules/dojo`

But experimentally I figured out that it happened because of missing dojo package in loader config. So I added this:

`      {
        name: "dojo",
        location: env.dojoRoot + "/dojo",
        lib: "."
      },`

And now it works as expected. 

Thanks again, 
issue can be closed.
